src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -179,6 +179,9 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
         private route: ActivatedRoute,
     ) {}
 
+    /**
+     * On init, finds the course, calculates exercise scores for each score type and groups exercises
```

Comment:
Here aswell it would be beneficial in my opinion to go more into detail, for example by what the exercises are grouped etc.